docs/zh-CN/sql-reference/sql-functions/conditional-functions/case.md

```diff
@@ -0,0 +1,59 @@
+---
+{
+    "title": "case",
+    "language": "zh-CN"
+}
+---
+
+<!-- 
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# case
+## description
+### Syntax
+
+`CASE a WHEN b THEN c [WHEN d THEN e]... [ELSE f] END`
+
+OR
+
+`CASE WHEN a THEN b [WHEN c THEN d]* [ELSE e] END`
+
```

Comment:
It is hard to understand a-e,  we need word like condition to express more things.

Reply:
OK